recipe/migrations/nodejs24.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for nodejs 24
+  kind: version
+  migration_number: 1
+migrator_ts: 1752132927.4207344
+nodejs:
+- '24'
```

```suggestion
- '24'
- '22'
```

C.f. previous migrations, e.g. #6110, #4782